### DISCUSSION TITLE
Does the "Outdated MMS Config Database" affect size limits on sent MMS? I cannot send a photo larger than 600KB

### DISCUSSION THREAD

**Discussion by bigrjsuto** [octoshrimpy/quik#104] [Q&A]
I figured I'd ask this here before adding a new issue in case it does.

Thanks.

**Comment by octoshrimpy** [comment]
I don't believe so. Did you make sure the settings for MMS size limit is not set to 600KB?

**Reply by VirginiaMaxwell** [reply]
I am having issues sending, I choose automatic and sometimes, uncompressed.  The only way images are sent if I retake the picture and send it within the message, attachments aren't working.
